hq/app/views/snyk/snyk.scala.html

```diff
@@ -0,0 +1,54 @@
+@import logic.SnykDisplay
+@import logic._
+@import model._
+
+@(results: List[SnykProjectIssues])(implicit assets: AssetsFinder)
+
+@main(List("Snyk")) { @* Header *@
+    <div class="container">
+        <div class="row">
+            <h1 class="header center-on-small-only grey-text text-lighten-5">Security HQ</h1>
+            <div class="col s12 m9">
+                <h4 class="grey-text text-darken-4 center-on-small-only">Snyk Results</h4>
+            </div>
+        </div>
+    </div>
```

Comment:
There was a bit of a design change with the header and subheader, so the h1 should no longer be required. We will probably want something similar to the docs page subheader:

```
  <div class="hq-sub-header">
      <div class="container hq-sub-header__row">
          <div class="hq-sub-header__name">
              <h4 class="header light grey-text text-lighten-5">Snyk Results</h4>
          </div>
      </div>
  </div>
```

Not a blocker, can be target for subsequent PR 😄